content/pages/index.rst

```diff
@@ -8,15 +8,11 @@ home
 .. raw:: html
     
     <h2 class="text-2xl text-center mb-5">Upcoming Events</h2>
-    <div class="flex flex-row flex-wrap text-center">
-        <div class="md:w-1/2 w-full flex-col space-y-1 items-center">
-            <h3 class="text-lg">scisprint 2024 October in Hsinchu</h3>
+    <div class="flex flex-row flex-wrap text-center items-center justify-center">
```

Comment:
Make the single event align to center.
<img width="1512" alt="image" src="https://github.com/user-attachments/assets/0a40bde0-f30b-4ff9-a339-e53ab98c828d">